hangzhounet/values.yaml

```
@@ -0,0 +1,149 @@
+    key: edpkvMWHXeETjH4i2h2ngkG2y6aVdgtW99pQ3SuHJoWNEPKANzVRLT
+    type: public
+    is_bootstrap_baker_account: false
+    bootstrap_balance: "5000000000000"
+  tz1THUNA:
+    # tz1THUNARo58aD5YdNGYPnWNnvd8yHPrMdsF 
+    # Romain's faucet
+    key: edpkuh3WSwziyd6nXqDXeh48vZHp5ETxFd43EdkbzhmW98BvuLXK81
+    type: public
+    is_bootstrap_baker_account: false
+    bootstrap_balance: "500000000000000"
+  oxheadbaker:
+    # Oxhead Alpha's baker hosted in teztnets.xyz infra
+    # tz1foXHgRzdYdaLgX6XhpZGxbBv42LZ6ubvE
+    bootstrap_balance: "5000000000000"
+    is_bootstrap_baker_account: true
+    type: secret
+    # key injected by pulumi
+  oxheadfree:
+    # an account with a lot of money in it
+    # tz1foxFdz2ieSj8P9YxKYBTEqYbgFXXEeiQY
+    key: edpkubSgCoMjB2a6vPWRNRHfGa6HUFfFyEUKoEQhWC8osnav8QzBdJ
+    bootstrap_balance: "5000000000000000"
+    is_bootstrap_baker_account: false
+    type: public
+
+node_config_network:
+  activation_account_name: oxheadbaker
+  chain_name: TEZOS_HANGZHOUNET_2021-09-22T15:00:00Z
+  genesis:
+    block: BLockGenesisGenesisGenesisGenesisGenesisd4299hBGVoU
+    protocol: PtYuensgYBb3G3x1hLLbCmcav8ue8Kyd2khADcL5LsT5R1hcXex
+    timestamp: "2021-09-22T15:00:00Z"
+  user_activated_upgrades:
+    - level: 4095
+      replacement_protocol: PtHangzHogokSuiMHemCuowEavgYTP8J5qQ9fQS793MHYFpCY3r
+
+bootstrap_peers:
+  - hangzhounet.smartpy.io
+  - hangzhounet.tezos.co.il
+  - hangzhounet.kaml.fr
+
+images:
+  octez: tezos/tezos:v11.0-rc1
+
+nodes:
+  tezos-baking-node:
+    instances:
+      - bake_using_accounts:
+          - oxheadbaker
+        config:
+          shell:
+            history_mode: archive
+          rpc:
+            cors-headers: [ "content-type" ]
+            cors-origin: [ "*" ]
+        is_bootstrap_node: true
+    runs:
+      - baker
+      - endorser
+      - octez_node
+    storage_size: 30Gi
+  # Overwrite default values.yaml tezos-node
+  tezos-node: null
+
+activation:
+  protocol_hash: PtGRANADsDU8R9daYKAgWnQYAJ64omN1o3KMGVCykShA97vQbvV
+  protocol_parameters:
+     preserved_cycles: 3
+     blocks_per_cycle: 4096
+     blocks_per_commitment: 32
+     blocks_per_roll_snapshot: 256
+     blocks_per_voting_period: 20480
+     time_between_blocks:
+       - "10"
+       - "20"
+     endorsers_per_block: 256
+     hard_gas_limit_per_operation: "1040000"
+     hard_gas_limit_per_block: "5200000"
+     proof_of_work_threshold: "70368744177663"
+     tokens_per_roll: "8000000000"
+     seed_nonce_revelation_tip: "125000"
+     origination_size: 257
+     block_security_deposit: "512000000"
+     endorsement_security_deposit: "64000000"
+     baking_reward_per_endorsement:
+       - "78125"
+       - "11719"
+     endorsement_reward:
+       - "78125"
+       - "52083"
+     cost_per_byte: "250"
+     hard_storage_limit_per_operation: "32768"
+     quorum_min: 2000
+     quorum_max: 7000
+     min_proposal_quorum: 500
+     initial_endorsers: 8
+     delay_per_missing_endorsement: "4"
```

```suggestion
     delay_per_missing_endorsement: "2"
```
(if you want a chain that goes twice faster)